packages/blockchain-wallet-v4/src/redux/kvStore/shapeShift/reducers.js

```diff
@@ -22,6 +23,9 @@ export default (state = INITIAL_STATE, action) => {
     case AT.FETCH_METADATA_SHAPESHIFT_FAILURE: {
       return Remote.Failure(payload)
     }
+    case AT.ADD_STATE_METADATA_SHAPESHIFT: {
+      return set(compose(mapped, value, lensPath(['USAState', 'Code'])), payload.usState, state)
```

Comment:
`USAState` must be saved as a `string` so iOS and Android can correctly parse. Currently it's saved as {Code: 'State'}